tests/python_package_test/test_engine.py

```diff
@@ -1417,13 +1417,10 @@ def test_get_split_value_histogram(self):
         self.assertRaises(lgb.basic.LightGBMError, gbm.get_split_value_histogram, 2)
 
     def test_early_stopping_for_only_first_metric(self):
+        np.random.seed(71)
```

Comment:
Reset seed before exit from this function.
https://github.com/microsoft/LightGBM/blob/b1e5a843ab57c64b01c8896ef3cda6fdf907d417/tests/python_package_test/test_engine.py#L627